.github/workflows/build.yml

```
@@ -13,7 +13,7 @@ on:
       - trying
       - "renovate/**"
     tags:
-      - '[0-9][0-9].[0-9]+.[0-9]+'
+      - '[0-9][0-9].[0-9]+.[0-9]+(-rc[0-9]+)?'
```

It seems like the `(...)?` isn't working.

```suggestion
      - '[0-9][0-9].[0-9]+.[0-9]+'
      - '[0-9][0-9].[0-9]+.[0-9]+-rc[0-9]+'
```